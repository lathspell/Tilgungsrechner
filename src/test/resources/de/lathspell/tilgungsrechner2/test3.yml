# Darlehensbetrag               60.000,00€
# Auszahlungsdatum              01.04.2016
# Häufigkeit der Ratenzahlung   monatlich
# Dauer der Sollzinsbindung     20 Jahre
# Gebundener Sollzinssatz       1,23% pro Jahr
# Anfängliche Tilgung           2,00%
# Höhe der Rate                 161,50€
# Getilgter Betrag zum Ende der Sollzinsbindung     51.160,58€
# Restschuld zum Ende der Sollzinsbindung           8.839,42€
# Summe der geleisteten Zinszahlungen               8.774,42€
# Rate bei Sollzinsanstieg von 2% am Ende der Sollzinsbindung       276,23€
# Darlehensdauer bei gleich bleibendem Zins                         Feb. 2039
# Gesamtlaufzeit des Darlehens                      22 Jahre 11 Monate
---
months:
    "2016-04":
        darlehensbetrag: 60000
        sollzins: 1.23
        monatsrate: 161.50
        auszahlung: 60000
    "2020-01":
        sondertilgung: 10000
    "2020-03":
        monatsrate: 136.50
    "2025-08":
        monatsrate: 261.50
    "2036-04":
        zinsbindung: true
    