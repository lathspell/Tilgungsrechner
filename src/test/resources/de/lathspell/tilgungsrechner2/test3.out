# test3
Tilgungsplan
Periode 	Rate (EUR) 	Zinsanteil (EUR) 	Tilgungsanteil (EUR) 	Restschuld am Periodenende (EUR)
Apr 2016           161,50              61,50                 100,00                 59.900,00
Mai 2016           161,50              61,40                 100,10                 59.799,90
Jun 2016           161,50              61,29                 100,21                 59.699,69
Jul 2016           161,50              61,19                 100,31                 59.599,38
Aug 2016           161,50              61,09                 100,41                 59.498,97
Sep 2016           161,50              60,99                 100,51                 59.398,46
Okt 2016           161,50              60,88                 100,62                 59.297,84
Nov 2016           161,50              60,78                 100,72                 59.197,12
Dez 2016           161,50              60,68                 100,82                 59.096,30
Jan 2017           161,50              60,57                 100,93                 58.995,37
Feb 2017           161,50              60,47                 101,03                 58.894,34
Mär 2017           161,50              60,37                 101,13                 58.793,21
Apr 2017           161,50              60,26                 101,24                 58.691,97
Mai 2017           161,50              60,16                 101,34                 58.590,63
Jun 2017           161,50              60,06                 101,44                 58.489,19
Jul 2017           161,50              59,95                 101,55                 58.387,64
Aug 2017           161,50              59,85                 101,65                 58.285,99
Sep 2017           161,50              59,74                 101,76                 58.184,23
Okt 2017           161,50              59,64                 101,86                 58.082,37
Nov 2017           161,50              59,53                 101,97                 57.980,40
Dez 2017           161,50              59,43                 102,07                 57.878,33
Jan 2018           161,50              59,33                 102,17                 57.776,16
Feb 2018           161,50              59,22                 102,28                 57.673,88
Mär 2018           161,50              59,12                 102,38                 57.571,50
Apr 2018           161,50              59,01                 102,49                 57.469,01
Mai 2018           161,50              58,91                 102,59                 57.366,42
Jun 2018           161,50              58,80                 102,70                 57.263,72
Jul 2018           161,50              58,70                 102,80                 57.160,92
Aug 2018           161,50              58,59                 102,91                 57.058,01
Sep 2018           161,50              58,48                 103,02                 56.954,99
Okt 2018           161,50              58,38                 103,12                 56.851,87
Nov 2018           161,50              58,27                 103,23                 56.748,64
Dez 2018           161,50              58,17                 103,33                 56.645,31
Jan 2019           161,50              58,06                 103,44                 56.541,87
Feb 2019           161,50              57,96                 103,54                 56.438,33
Mär 2019           161,50              57,85                 103,65                 56.334,68
Apr 2019           161,50              57,74                 103,76                 56.230,92
Mai 2019           161,50              57,64                 103,86                 56.127,06
Jun 2019           161,50              57,53                 103,97                 56.023,09
Jul 2019           161,50              57,42                 104,08                 55.919,01
Aug 2019           161,50              57,32                 104,18                 55.814,83
Sep 2019           161,50              57,21                 104,29                 55.710,54
Okt 2019           161,50              57,10                 104,40                 55.606,14
Nov 2019           161,50              57,00                 104,50                 55.501,64
Dez 2019           161,50              56,89                 104,61                 55.397,03
>> Sondertilgung 10.000,00 EUR nach Periode Dez 2019
Jan 2020           161,50              46,53                 114,97                 45.282,06
Feb 2020           161,50              46,41                 115,09                 45.166,97
>> Neuer Tilgungssatz 1,50 % ab Mär 2020 (neue Rate 136,50)
Mär 2020           136,50              46,30                  90,20                 45.076,77
Apr 2020           136,50              46,20                  90,30                 44.986,47
Mai 2020           136,50              46,11                  90,39                 44.896,08
Jun 2020           136,50              46,02                  90,48                 44.805,60
Jul 2020           136,50              45,93                  90,57                 44.715,03
Aug 2020           136,50              45,83                  90,67                 44.624,36
Sep 2020           136,50              45,74                  90,76                 44.533,60
Okt 2020           136,50              45,65                  90,85                 44.442,75
Nov 2020           136,50              45,55                  90,95                 44.351,80
Dez 2020           136,50              45,46                  91,04                 44.260,76
Jan 2021           136,50              45,37                  91,13                 44.169,63
Feb 2021           136,50              45,27                  91,23                 44.078,40
Mär 2021           136,50              45,18                  91,32                 43.987,08
Apr 2021           136,50              45,09                  91,41                 43.895,67
Mai 2021           136,50              44,99                  91,51                 43.804,16
Jun 2021           136,50              44,90                  91,60                 43.712,56
Jul 2021           136,50              44,81                  91,69                 43.620,87
Aug 2021           136,50              44,71                  91,79                 43.529,08
Sep 2021           136,50              44,62                  91,88                 43.437,20
Okt 2021           136,50              44,52                  91,98                 43.345,22
Nov 2021           136,50              44,43                  92,07                 43.253,15
Dez 2021           136,50              44,33                  92,17                 43.160,98
Jan 2022           136,50              44,24                  92,26                 43.068,72
Feb 2022           136,50              44,15                  92,35                 42.976,37
Mär 2022           136,50              44,05                  92,45                 42.883,92
Apr 2022           136,50              43,96                  92,54                 42.791,38
Mai 2022           136,50              43,86                  92,64                 42.698,74
Jun 2022           136,50              43,77                  92,73                 42.606,01
Jul 2022           136,50              43,67                  92,83                 42.513,18
Aug 2022           136,50              43,58                  92,92                 42.420,26
Sep 2022           136,50              43,48                  93,02                 42.327,24
Okt 2022           136,50              43,39                  93,11                 42.234,13
Nov 2022           136,50              43,29                  93,21                 42.140,92
Dez 2022           136,50              43,19                  93,31                 42.047,61
Jan 2023           136,50              43,10                  93,40                 41.954,21
Feb 2023           136,50              43,00                  93,50                 41.860,71
Mär 2023           136,50              42,91                  93,59                 41.767,12
Apr 2023           136,50              42,81                  93,69                 41.673,43
Mai 2023           136,50              42,72                  93,78                 41.579,65
Jun 2023           136,50              42,62                  93,88                 41.485,77
Jul 2023           136,50              42,52                  93,98                 41.391,79
Aug 2023           136,50              42,43                  94,07                 41.297,72
Sep 2023           136,50              42,33                  94,17                 41.203,55
Okt 2023           136,50              42,23                  94,27                 41.109,28
Nov 2023           136,50              42,14                  94,36                 41.014,92
Dez 2023           136,50              42,04                  94,46                 40.920,46
Jan 2024           136,50              41,94                  94,56                 40.825,90
Feb 2024           136,50              41,85                  94,65                 40.731,25
Mär 2024           136,50              41,75                  94,75                 40.636,50
Apr 2024           136,50              41,65                  94,85                 40.541,65
Mai 2024           136,50              41,56                  94,94                 40.446,71
Jun 2024           136,50              41,46                  95,04                 40.351,67
Jul 2024           136,50              41,36                  95,14                 40.256,53
Aug 2024           136,50              41,26                  95,24                 40.161,29
Sep 2024           136,50              41,17                  95,33                 40.065,96
Okt 2024           136,50              41,07                  95,43                 39.970,53
Nov 2024           136,50              40,97                  95,53                 39.875,00
Dez 2024           136,50              40,87                  95,63                 39.779,37
Jan 2025           136,50              40,77                  95,73                 39.683,64
Feb 2025           136,50              40,68                  95,82                 39.587,82
Mär 2025           136,50              40,58                  95,92                 39.491,90
Apr 2025           136,50              40,48                  96,02                 39.395,88
Mai 2025           136,50              40,38                  96,12                 39.299,76
Jun 2025           136,50              40,28                  96,22                 39.203,54
Jul 2025           136,50              40,18                  96,32                 39.107,22
>> Neuer Tilgungssatz 4,00 % ab Aug 2025 (neue Rate 261,50)
Aug 2025           261,50              40,08                 221,42                 38.885,80
Sep 2025           261,50              39,86                 221,64                 38.664,16
Okt 2025           261,50              39,63                 221,87                 38.442,29
Nov 2025           261,50              39,40                 222,10                 38.220,19
Dez 2025           261,50              39,18                 222,32                 37.997,87
Jan 2026           261,50              38,95                 222,55                 37.775,32
Feb 2026           261,50              38,72                 222,78                 37.552,54
Mär 2026           261,50              38,49                 223,01                 37.329,53
Apr 2026           261,50              38,26                 223,24                 37.106,29
Mai 2026           261,50              38,03                 223,47                 36.882,82
Jun 2026           261,50              37,80                 223,70                 36.659,12
Jul 2026           261,50              37,58                 223,92                 36.435,20
Aug 2026           261,50              37,35                 224,15                 36.211,05
Sep 2026           261,50              37,12                 224,38                 35.986,67
Okt 2026           261,50              36,89                 224,61                 35.762,06
Nov 2026           261,50              36,66                 224,84                 35.537,22
Dez 2026           261,50              36,43                 225,07                 35.312,15
Jan 2027           261,50              36,19                 225,31                 35.086,84
Feb 2027           261,50              35,96                 225,54                 34.861,30
Mär 2027           261,50              35,73                 225,77                 34.635,53
Apr 2027           261,50              35,50                 226,00                 34.409,53
Mai 2027           261,50              35,27                 226,23                 34.183,30
Jun 2027           261,50              35,04                 226,46                 33.956,84
Jul 2027           261,50              34,81                 226,69                 33.730,15
Aug 2027           261,50              34,57                 226,93                 33.503,22
Sep 2027           261,50              34,34                 227,16                 33.276,06
Okt 2027           261,50              34,11                 227,39                 33.048,67
Nov 2027           261,50              33,87                 227,63                 32.821,04
Dez 2027           261,50              33,64                 227,86                 32.593,18
Jan 2028           261,50              33,41                 228,09                 32.365,09
Feb 2028           261,50              33,17                 228,33                 32.136,76
Mär 2028           261,50              32,94                 228,56                 31.908,20
Apr 2028           261,50              32,71                 228,79                 31.679,41
Mai 2028           261,50              32,47                 229,03                 31.450,38
Jun 2028           261,50              32,24                 229,26                 31.221,12
Jul 2028           261,50              32,00                 229,50                 30.991,62
Aug 2028           261,50              31,77                 229,73                 30.761,89
Sep 2028           261,50              31,53                 229,97                 30.531,92
Okt 2028           261,50              31,30                 230,20                 30.301,72
Nov 2028           261,50              31,06                 230,44                 30.071,28
Dez 2028           261,50              30,82                 230,68                 29.840,60
Jan 2029           261,50              30,59                 230,91                 29.609,69
Feb 2029           261,50              30,35                 231,15                 29.378,54
Mär 2029           261,50              30,11                 231,39                 29.147,15
Apr 2029           261,50              29,88                 231,62                 28.915,53
Mai 2029           261,50              29,64                 231,86                 28.683,67
Jun 2029           261,50              29,40                 232,10                 28.451,57
Jul 2029           261,50              29,16                 232,34                 28.219,23
Aug 2029           261,50              28,92                 232,58                 27.986,65
Sep 2029           261,50              28,69                 232,81                 27.753,84
Okt 2029           261,50              28,45                 233,05                 27.520,79
Nov 2029           261,50              28,21                 233,29                 27.287,50
Dez 2029           261,50              27,97                 233,53                 27.053,97
Jan 2030           261,50              27,73                 233,77                 26.820,20
Feb 2030           261,50              27,49                 234,01                 26.586,19
Mär 2030           261,50              27,25                 234,25                 26.351,94
Apr 2030           261,50              27,01                 234,49                 26.117,45
Mai 2030           261,50              26,77                 234,73                 25.882,72
Jun 2030           261,50              26,53                 234,97                 25.647,75
Jul 2030           261,50              26,29                 235,21                 25.412,54
Aug 2030           261,50              26,05                 235,45                 25.177,09
Sep 2030           261,50              25,81                 235,69                 24.941,40
Okt 2030           261,50              25,56                 235,94                 24.705,46
Nov 2030           261,50              25,32                 236,18                 24.469,28
Dez 2030           261,50              25,08                 236,42                 24.232,86
Jan 2031           261,50              24,84                 236,66                 23.996,20
Feb 2031           261,50              24,60                 236,90                 23.759,30
Mär 2031           261,50              24,35                 237,15                 23.522,15
Apr 2031           261,50              24,11                 237,39                 23.284,76
Mai 2031           261,50              23,87                 237,63                 23.047,13
Jun 2031           261,50              23,62                 237,88                 22.809,25
Jul 2031           261,50              23,38                 238,12                 22.571,13
Aug 2031           261,50              23,14                 238,36                 22.332,77
Sep 2031           261,50              22,89                 238,61                 22.094,16
Okt 2031           261,50              22,65                 238,85                 21.855,31
Nov 2031           261,50              22,40                 239,10                 21.616,21
Dez 2031           261,50              22,16                 239,34                 21.376,87
Jan 2032           261,50              21,91                 239,59                 21.137,28
Feb 2032           261,50              21,67                 239,83                 20.897,45
Mär 2032           261,50              21,42                 240,08                 20.657,37
Apr 2032           261,50              21,17                 240,33                 20.417,04
Mai 2032           261,50              20,93                 240,57                 20.176,47
Jun 2032           261,50              20,68                 240,82                 19.935,65
Jul 2032           261,50              20,43                 241,07                 19.694,58
Aug 2032           261,50              20,19                 241,31                 19.453,27
Sep 2032           261,50              19,94                 241,56                 19.211,71
Okt 2032           261,50              19,69                 241,81                 18.969,90
Nov 2032           261,50              19,44                 242,06                 18.727,84
Dez 2032           261,50              19,20                 242,30                 18.485,54
Jan 2033           261,50              18,95                 242,55                 18.242,99
Feb 2033           261,50              18,70                 242,80                 18.000,19
Mär 2033           261,50              18,45                 243,05                 17.757,14
Apr 2033           261,50              18,20                 243,30                 17.513,84
Mai 2033           261,50              17,95                 243,55                 17.270,29
Jun 2033           261,50              17,70                 243,80                 17.026,49
Jul 2033           261,50              17,45                 244,05                 16.782,44
Aug 2033           261,50              17,20                 244,30                 16.538,14
Sep 2033           261,50              16,95                 244,55                 16.293,59
Okt 2033           261,50              16,70                 244,80                 16.048,79
Nov 2033           261,50              16,45                 245,05                 15.803,74
Dez 2033           261,50              16,20                 245,30                 15.558,44
Jan 2034           261,50              15,95                 245,55                 15.312,89
Feb 2034           261,50              15,70                 245,80                 15.067,09
Mär 2034           261,50              15,44                 246,06                 14.821,03
Apr 2034           261,50              15,19                 246,31                 14.574,72
Mai 2034           261,50              14,94                 246,56                 14.328,16
Jun 2034           261,50              14,69                 246,81                 14.081,35
Jul 2034           261,50              14,43                 247,07                 13.834,28
Aug 2034           261,50              14,18                 247,32                 13.586,96
Sep 2034           261,50              13,93                 247,57                 13.339,39
Okt 2034           261,50              13,67                 247,83                 13.091,56
Nov 2034           261,50              13,42                 248,08                 12.843,48
Dez 2034           261,50              13,16                 248,34                 12.595,14
Jan 2035           261,50              12,91                 248,59                 12.346,55
Feb 2035           261,50              12,66                 248,84                 12.097,71
Mär 2035           261,50              12,40                 249,10                 11.848,61
Apr 2035           261,50              12,14                 249,36                 11.599,25
Mai 2035           261,50              11,89                 249,61                 11.349,64
Jun 2035           261,50              11,63                 249,87                 11.099,77
Jul 2035           261,50              11,38                 250,12                 10.849,65
Aug 2035           261,50              11,12                 250,38                 10.599,27
Sep 2035           261,50              10,86                 250,64                 10.348,63
Okt 2035           261,50              10,61                 250,89                 10.097,74
Nov 2035           261,50              10,35                 251,15                  9.846,59
Dez 2035           261,50              10,09                 251,41                  9.595,18
Jan 2036           261,50               9,84                 251,66                  9.343,52
Feb 2036           261,50               9,58                 251,92                  9.091,60
Mär 2036           261,50               9,32                 252,18                  8.839,42
Apr 2036           261,50               9,06                 252,44                  8.586,98
Mai 2036           261,50               8,80                 252,70                  8.334,28
Jun 2036           261,50               8,54                 252,96                  8.081,32
Jul 2036           261,50               8,28                 253,22                  7.828,10
Aug 2036           261,50               8,02                 253,48                  7.574,62
Sep 2036           261,50               7,76                 253,74                  7.320,88
Okt 2036           261,50               7,50                 254,00                  7.066,88
Nov 2036           261,50               7,24                 254,26                  6.812,62
Dez 2036           261,50               6,98                 254,52                  6.558,10
Jan 2037           261,50               6,72                 254,78                  6.303,32
Feb 2037           261,50               6,46                 255,04                  6.048,28
Mär 2037           261,50               6,20                 255,30                  5.792,98
Apr 2037           261,50               5,94                 255,56                  5.537,42
Mai 2037           261,50               5,68                 255,82                  5.281,60
Jun 2037           261,50               5,41                 256,09                  5.025,51
Jul 2037           261,50               5,15                 256,35                  4.769,16
Aug 2037           261,50               4,89                 256,61                  4.512,55
Sep 2037           261,50               4,63                 256,87                  4.255,68
Okt 2037           261,50               4,36                 257,14                  3.998,54
Nov 2037           261,50               4,10                 257,40                  3.741,14
Dez 2037           261,50               3,83                 257,67                  3.483,47
Jan 2038           261,50               3,57                 257,93                  3.225,54
Feb 2038           261,50               3,31                 258,19                  2.967,35
Mär 2038           261,50               3,04                 258,46                  2.708,89
Apr 2038           261,50               2,78                 258,72                  2.450,17
Mai 2038           261,50               2,51                 258,99                  2.191,18
Jun 2038           261,50               2,25                 259,25                  1.931,93
Jul 2038           261,50               1,98                 259,52                  1.672,41
Aug 2038           261,50               1,71                 259,79                  1.412,62
Sep 2038           261,50               1,45                 260,05                  1.152,57
Okt 2038           261,50               1,18                 260,32                    892,25
Nov 2038           261,50               0,91                 260,59                    631,66
Dez 2038           261,50               0,65                 260,85                    370,81
Jan 2039           261,50               0,38                 261,12                    109,69
Feb 2039           109,80               0,11                 109,69                      0,00
